core/services/relay/evm/mercury/config_poller.go

```diff
@@ -138,10 +147,13 @@ func (lp *ConfigPoller) LatestConfigDetails(ctx context.Context) (changedInBlock
 
 // LatestConfig returns the latest config from the logs on a certain block
 func (lp *ConfigPoller) LatestConfig(ctx context.Context, changedInBlock uint64) (ocrtypes.ContractConfig, error) {
-	lgs, err := lp.destChainLogPoller.Logs(int64(changedInBlock), int64(changedInBlock), FeedScopedConfigSet, lp.addr, pg.WithParentCtx(ctx))
+	lgs, err := lp.destChainLogPoller.IndexedLogsByBlockRange(int64(changedInBlock), int64(changedInBlock), FeedScopedConfigSet, lp.addr, feedIdTopicIndex, []common.Hash{*lp.feedId}, pg.WithParentCtx(ctx))
```

Comment:
Did this previously return error if there were no logs?

Reply:
It was being handled by the `sql.ErrNoRows` but that's no longer being triggered, so I added a check in case there are no logs.

Reply:
Yeah, so previously this would return error if there were no logs but now it returns nil?

Reply:
Hmm, good question, this never checked for the lack of logs... So whatever the error returned from the ORM was, it was the resulted error, the `sql.ErrNoRows` happens on the `LatestConfigDetails` call. So I wonder if we should just return an error now, to match the previous implementation...